homeassistant/components/ezviz/select.py

```diff
@@ -0,0 +1,83 @@
+"""Support for EZVIZ select controls."""
+from __future__ import annotations
+
+from pyezviz.constants import SoundMode
+from pyezviz.exceptions import HTTPError, PyEzvizError
+
+from homeassistant.components.select import SelectEntity, SelectEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DATA_COORDINATOR, DOMAIN
+from .coordinator import EzvizDataUpdateCoordinator
+from .entity import EzvizEntity
+
+PARALLEL_UPDATES = 1
+
+SELECT_TYPES = SelectEntityDescription(
+    key="alarm_sound_mod",
+    name="Warning sound",
+    icon="mdi:alarm",
+    entity_category=EntityCategory.CONFIG,
+    options=["soft", "intensive", "silent"],
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up EZVIZ select entities based on a config entry."""
+    coordinator: EzvizDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id][
+        DATA_COORDINATOR
+    ]
+
+    async_add_entities(
+        EzvizSensor(coordinator, camera, entity, SELECT_TYPES)
+        for camera in coordinator.data
+        for entity, value in coordinator.data[camera].items()
+        if entity in SELECT_TYPES.key
```

Comment:
Doesn't seem right. Don't think `entity` is the same as the key in `SELECT_TYPES`

Reply:
Thanks. Found a better way to only create entity if camera supports the feature.